ocean_provider/utils/address.py

```diff
@@ -27,3 +31,14 @@ def get_contract_address(
         for chain_addresses in address_json.values()
         if chain_addresses["chainId"] == chain_id
     )
+
+
+def get_provider_fee_token(chain_id):
+    fee_token = os.environ.get("PROVIDER_FEE_TOKEN", get_ocean_address(chain_id))
+    if not fee_token:
```

Comment:
i suggest using the ternary: `return fee_token if fee_token else BLACK_HOLE_ADDRESS`.

Reply:
coming soon